.drone.yml

```diff
@@ -1,129 +1,99 @@
 pipeline:
   test_build:
-    image: docker:1.11
+    image: docker:17.09.0-ce
     environment:
-      - DOCKER_HOST=tcp://127.0.0.1:2375
+      - DOCKER_HOST=tcp://172.17.0.1:2375
     commands:
-      - apk add --no-cache bash sed grep bc coreutils
+      - apk add --no-cache bash sed grep bc coreutils curl
       - ./ci-build.sh
     when:
-      event: [push, pull_request, tag]
+      event: [push, pull_request]
   
   build_clamav:
-    image: docker:1.11
+    image: docker:17.09.0-ce
     environment:
-      - DOCKER_HOST=tcp://127.0.0.1:2375
+      - DOCKER_HOST=tcp://172.17.0.1:2375
     commands:
-      - docker build -t clamav .
+      - docker build -t clamav:$${DRONE_COMMIT_SHA} .
     when:
-      branch: [master, refs/tags/*]
+      branch: master
```

Comment:
I'm not sure how drone reports branch on tag events... Previously pipeline used `branch: [master, refs/tags/*]` - something to check probably.

Reply:
When you tag it technically counts as a push to a branch called `refs/tags/<tag>` and drone counts that as a tag, not a push. (See the payload with id `b6a320d8-c562-11e7-8fda-ade8bfc449a1`)

Although I did try tagging while this was there but it still didn't trigger when I tried.